src/test/suite/ruby.test.ts

```diff
@@ -0,0 +1,65 @@
+import * as assert from "assert";
+
+import { before, after } from "mocha";
+import * as vscode from "vscode";
+
+import { Ruby } from "../../ruby";
+
+suite("Ruby environment activation", () => {
+  let ruby: Ruby;
+  const configuration = vscode.workspace.getConfiguration("rubyLsp");
+  const currentManager = configuration.get("rubyVersionManager")!;
+
+  before(async () => {
+    await configuration.update("rubyVersionManager", "none", true, true);
+  });
+
+  after(async () => {
+    await configuration.update(
+      "rubyVersionManager",
+      currentManager,
+      true,
+      true
+    );
+  });
+
+  test("Activate fetches Ruby information when outside of Ruby LSP", async () => {
+    ruby = new Ruby("fake/some/project");
+    await ruby.activateRuby();
+
+    const [major, minor, _patch] = ruby.rubyVersion!.split(".").map(Number);
+    const supportsYjit = ruby.yjitEnabled && [major, minor] >= [3, 2];
+
+    assert.ok(ruby.rubyVersion, "Expected Ruby version to be set");
+    assert.strictEqual(
+      ruby.supportsYjit,
+      supportsYjit,
```

Comment:
If we're only testing against Ruby 3.2, isn't this always `true`?
If we want to cover other scenarios, perhaps we should have a Ruby version matrix?

Reply:
Thanks for simplifying it 👍